channels/chan_usbradio.c

```diff
@@ -2096,6 +2135,12 @@ static struct ast_frame *usbradio_read(struct ast_channel *c)
 	}
 #endif
 
+	/* TBR - below appears to be an attempt to match levels to the original CM108
+	 * IC which has been out of production for over 10 years. Scaling audio to 110%
+	 * will result in clipping! Any adjustments for CM1xxx IC gain differences
+	 * should be made in the mixer settings, not in the audio stream itself.
+	 */
+#if 0
```

Comment:
Please see my comments in chan_simpleusb.c re: concerns about this #if ... #endif block.

Reply:
Bug report opened: https://github.com/AllStarLink/app_rpt/issues/399